clients/src/main/resources/common/message/ShareFetchResponse.json

```diff
@@ -67,7 +67,7 @@
           { "name": "LeaderEpoch", "type": "int32", "versions": "0+",
             "about": "The latest known leader epoch." }
         ]},
-        { "name": "Records", "type": "records", "versions": "0+", "nullableVersions": "0+", "about": "The record data." },
+        { "name": "Records", "type": "records", "versions": "0+", "about": "The record data." },
```

Comment:
Technically, that would be `"nullableVersions": "0"` I suppose. It was nullable in v0. We are now in v1.

Reply:
the v0 is not available for now, so it should be fine to remove `nullableVersions`?

Reply:
Well, yes, unless we are looking to have the JSON file document what was previously there too for v0. Otherwise, we could entirely remove all mention of v0, which we did not do.